### PR TITLE
Disable collector in phoebe.spec

### PR DESCRIPTION
**Pull request by shunghsiyu**
This is based on top of @asmorodskyi work in #45.

I removed Python-related dependencies and disabled the collector in `phoebe.spec`. Hopefully this is all that's left to do to get the CI happy.

### CODE REVIEW

File: dist/rpm/phoebe.spec [hunk 1]
```diff
@@ -54,7 +51,7 @@ to offer eventually the best possible setup.
 %build
 # export build flags manually if %%set_build_flags is not defined
 %{?!set_build_flags:export CFLAGS="%{optflags}"; export LDFLAGS="${RPM_LD_FLAGS}"}
-%meson -Dwith_tests=false -Dsystemd_system_unit_dir=%{_unitdir}
+%meson -Dwith_tests=false -Dsystemd_system_unit_dir=%{_unitdir} -Dcollector=disabled
```
Comment (dcermak) [inline dist/rpm/phoebe.spec:54]: Adding this option makes no difference for the resulting rpm (it's just faster to build and has less build time dependencies), so I'm in favor of this.

**Review by dcermak — APPROVED** [review]

**Comment by shunghsiyu** [comment]
Hmm...I think [build_rpm failure on Leap 15.2](https://github.com/SUSE/phoebe/runs/2446288091?check_suite_focus=true) may be due to the use of [`disabler`](https://mesonbuild.com/Reference-manual.html#disabler)?

```
DEBUG: Program python3 found: NO (disabled by: collector )
DEBUG: scripts/meson.build:1:0: ERROR: Module returned a value of unknown type.
DEBUG: A full log can be found at /builddir/build/BUILD/phoebe-0.1.1/build/meson-logs/meson-log.txt
DEBUG: error: Bad exit status from /var/tmp/rpm-tmp.ywxUuq (%build)
DEBUG:     Bad exit status from /var/tmp/rpm-tmp.ywxUuq (%build)
DEBUG: RPM build errors:
DEBUG: Child return code was: 1
INFO: EXCEPTION: [Error()]
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/mockbuild/trace_decorator.py", line 93, in trace
    result = func(*args, **kw)
  File "/usr/lib/python3.9/site-packages/mockbuild/util.py", line 600, in do_with_status
    raise exception.Error("Command failed: \n # %s\n%s" % (command, output), child.returncode)
```

Let me look into that.

**Comment by mvarlese** [comment]
> Hmm...I think [build_rpm failure on Leap 15.2](https://github.com/SUSE/phoebe/runs/2446288091?check_suite_focus=true) may be due to the use of [`disabler`](https://mesonbuild.com/Reference-manual.html#disabler)?
> 
> ```
> DEBUG: Program python3 found: NO (disabled by: collector )
> DEBUG: scripts/meson.build:1:0: ERROR: Module returned a value of unknown type.
> DEBUG: A full log can be found at /builddir/build/BUILD/phoebe-0.1.1/build/meson-logs/meson-log.txt
> DEBUG: error: Bad exit status from /var/tmp/rpm-tmp.ywxUuq (%build)
> DEBUG:     Bad exit status from /var/tmp/rpm-tmp.ywxUuq (%build)
> DEBUG: RPM build errors:
> DEBUG: Child return code was: 1
> INFO: EXCEPTION: [Error()]
> Traceback (most recent call last):
>   File "/usr/lib/python3.9/site-packages/mockbuild/trace_decorator.py", line 93, in trace
>     result = func(*args, **kw)
>   File "/usr/lib/python3.9/site-packages/mockbuild/util.py", line 600, in do_with_status
>     raise exception.Error("Command failed: \n # %s\n%s" % (command, output), child.returncode)
> ```
> 
> Let me look into that.

@shunghsiyu do you want to first look at the leap15.2 failure or are you happy to have your patch merged nonetheless?

**Comment by shunghsiyu** [comment]
With workaround in d52f48232ea81f2fbeb08dd0fc14b51140d20a36 the failure on Leap 15.2 does not show up anymore :smile: 

Let me go ahead and merge this PR

**Pull request merged**